### ISSUE TITLE
Choose your hash algorithm

### ISSUE THREAD

**Issue by hmonglee** [ETSGlobal/ETSPaymentOgoneBundle#69]
Hello,

default hash algorithm is set to sha1, but we can choose between sha1, sha256 and sha512. 

I think it was good to can set hash algorithm.

**Comment by jantajov** [comment]
I agree it is definitely a good idea since the SHA1 not 100% safe anymore.
See: https://security.googleblog.com/2017/02/announcing-first-sha1-collision.html
